php/common.php

```diff
@@ -26,6 +25,23 @@
     array($avgSeedersPeriodOutdatedSeconds)
 );
 
+function get_webtlo_version()
+{
+    $webtlo = json_decode(file_get_contents(dirname(__FILE__) . '/../version.json'));
+    if (!isset($webtlo)) { 
+      return (object)[];
+    }
+
+    $webtlo->version_url = $webtlo->report = $webtlo->report_url = "";
+    if (isset($webtlo->version)) {
+        $webtlo->version_url = $webtlo->github . '/releases/tag/' . $webtlo->version;
+
+        $webtlo->report     = 'Версия TLO: [b]Web-TLO-' . $webtlo->version . '[/b]';
+        $webtlo->report_url = 'Версия TLO: [b]Web-TLO-[url='. $webtlo->version_url . ']' . $webtlo->version . '[/url][/b]';
+    }
+    return $webtlo;
+}
```

Comment:
In case of an absent file, PHP generates a lot of "absent property" messages. So that I would kindly propose a way with default values to be overridden if the file exists. In this suggestion I took into account my comment regarding naming and the comment from Orhideous, as I agree that the line should be still printed in reports.

```suggestion
function get_webtlo_version()
{
    $webtlo_version_defaults = array(
        'version' => '',
        'github' => '',
        'wiki' => '',
        'release' => '',
        'release_api' => '',
        'version_url' => '',
        'version_line' => 'Версия TLO: [b]Web-TLO-unknown[/b]',
        'version_line_url' => "Версия TLO: [b]Web-TLO-[url='#']unknown[/url][/b]"
    );
    $version_json_path = dirname(__FILE__) . '/../version.json';
    if (!file_exists($version_json_path)) {
        error_log('`version.json` not found! Make sure you copied all files from the repo.');
        return (object) $webtlo_version_defaults;
    }
    $version_json = json_decode(file_get_contents($version_json_path), true);
    $result = (object) array_merge($webtlo_version_defaults, $version_json);

    if (!empty($result->version)) {
        $result->version_url = $result->github . '/releases/tag/' . $result->version;
        $result->version_line     = 'Версия TLO: [b]Web-TLO-' . $result->version . '[/b]';
        $result->version_line_url = 'Версия TLO: [b]Web-TLO-[url='. $result->version_url . ']' . $result->version . '[/url][/b]';
    }
    return $result;
}
```

Please do not forget to rename variables `version_line` and `version_line_url` if the renaming proposal is accepted.